beacon-chain/state/types/types.go

```diff
@@ -110,6 +110,10 @@ const (
 	PreviousJustifiedCheckpoint
 	CurrentJustifiedCheckpoint
 	FinalizedCheckpoint
+	// State Fields Added in Altair.
+	InactivityScores
+	CurrentSyncCommittee
+	NextSyncCommittee
 )
 
 // State Fields Added in Altair.
```

Comment:
Seems confusing to why this comment is here.

Also why `iota + FinalizedCheckpoint + 1` didn't work?

Reply:
Fixed the comment, also `iota + FinalizedCheckpoint + 1` conflicts with the `iota` value previously declared. So now we only use it once.